draft-ietf-quic-recovery.md

```diff
@@ -601,8 +601,8 @@ occurs across all spaces to prevent excess load on the network.  For example,
 a timeout in the Initial packet number space doubles the length of the timeout
 in the Handshake packet number space.
 
-The life of a connection that is experiencing consecutive PTOs is limited by
-the endpoint's idle timeout.
+The time length of a connection that is experiencing consecutive PTOs is
+limited by the endpoint's idle timeout.
```

Comment:
```suggestion
The total length of time over which consecutive probes are sent is limited by the idle timeout.
```
?

Reply:
Sure, works for me.

Reply:
I'm honestly not sure we need this sentence, but I suspect it was added because someone asked for it?